sys/fido2/ctap/ctap.c

```diff
@@ -1792,12 +1793,15 @@ static int _ctap_decrypt_rk(ctap_resident_key_t *rk, ctap_cred_id_t *id)
 
 static int _write_state_to_flash(const ctap_state_t *state)
 {
-    return fido2_ctap_mem_write(state, CTAP_FLASH_STATE_PAGE, 0, CTAP_FLASH_STATE_SZ);
+    return fido2_ctap_mem_write(state,
```

Comment:
If I get it correctly, the "first" (available) page is always used for the state. Would be nice to leave a comment about that here and in `_read_state_from_flash`